tests/framework/callbacks/test_logger.py

```diff
@@ -151,6 +147,7 @@ def _test_logging(self, history):
             calls.append(call('loss', {'loss': h['loss'], 'val_loss': h['val_loss']}, h['epoch']))
             calls.append(call('lr', {'lr': 1e-3}, h['epoch']))
         self.writer.add_scalars.assert_has_calls(calls, any_order=True)
+        self.writer.add_scalars.assert_called_with('last-epoch', 10)
```

Comment:
Use an argument or a constant for the 10.

Reply:
done